RecoLocalTracker/SiPixelRecHits/python/PixelCPEGeneric_cfi.py

```diff
@@ -72,3 +72,16 @@
   Upgrade = cms.bool(True)
 )
 
+
+from Configuration.Eras.Modifier_phase2_tracker_postTDR_cff import phase2_tracker_postTDR
+phase2_tracker_postTDR.toModify(PixelCPEGenericESProducer, 
+  useLAWidthFromDB = False,
+  UseErrorsFromTemplates = False,
+  LoadTemplatesFromDB = False,
+  TruncatePixelCharge = False,
+  IrradiationBiasCorrection = False,
+  DoCosmics = False,
+  Upgrade = cms.bool(True),
+  lAOffset = cms.double(0.0459)
```

Comment:
only this ```lAOffset = 0.0459``` is needed, everything else is unnecessary copy-paste of the phase2_tracker mods

Reply:
@slava77 I am bit confused: should I define the phase2_tracker_postTDR mod as a clone of phase2_tracker mod and then add lAOffset?

Reply:
your current implementation is for phase2_tracker_postTDR to add changes on top of phase2_tracker.